### ISSUE TITLE
Rules and meta discussion here!

### ISSUE THREAD

**Issue by bernerbrau** [bernerbrau/AsTheWyrmTurns#1]
Not to be overly confusing by requiring you to look in multiple locations, but so as not to overwhelm the gameplay thread with meta talk. I'll also wiki FAQs for quick reference.

**Comment by kabobbob** [comment]
So, if we wanted to talk strategy about how Ash should hatchet the wyrm or the guy, that should happen here?

**Comment by ishaffer** [comment]
i was certainly thinking Ash should do something like that.  He's primed.  Doesn't James narrate his own fate?  isn't that the advantage of conceding over being taken out if I'm reading that right?

**Comment by kabobbob** [comment]
i'm thinking digs by could throw up some spell to incapacitate the guys

**Comment by kabobbob** [comment]
He typed concede, but he did edit it to be taken out, which is correct because he rolled and failed.

**Comment by bernerbrau** [comment]
So Bob, it's already been done as a Create Advantage for now, but for future reference I think casting a sleep spell would be better modeled as a mental Attack, which would have the effect of taking them out of the conflict by putting them to sleep.

**Comment by ishaffer** [comment]
So for succeeding with style on an attack action, do I get 2 free invokes on an aspect like with the "create an advantage" action, or do I decide to reduce the damage by 1 to get a boost?  I saw that latter rule in the Fate Accelerated rules (https://fate-srd.com/fate-accelerated/how-do-stuff-outcomes-actions-and-approaches) but wasn't sure if Fate Core is the same?

**Comment by bernerbrau** [comment]
https://fate-srd.com/fate-core/four-actions

**When you succeed with style on an attack,** it works like a normal success, but you also have the option to reduce the value of your hit by one to gain a boost as well.

**Comment by bernerbrau** [comment]
So basically, you could sacrifice +1 now for maybe +2 later.

**Comment by bernerbrau** [comment]
Point after, or two-point conversion. Take your pick.

**Comment by bernerbrau** [comment]
BTW, magic: https://fate-srd.com/fate-system-toolkit/magic

**Comment by bernerbrau** [comment]
By the way, THIS is an awesome example of how conflict in Fate is supposed to work. Highly chaotic and dynamic and in constant flux, each turn ratcheting up the dramatic tension. 

So with that in mind, don't forget: In Fate, the rules and mechanics are there to help you tell a good story. Therefore, whether or not you should be allowed to invoke an aspect is _directly_ tied to how well you can use that aspect to help you tell your part of the story.

**Comment by kabobbob** [comment]
BTW: i'm dying over "Axe in his face"!!!

**Comment by bernerbrau** [comment]
And he doesn't even "Start to panic" until some time **after** there's an "axe in his face"!

**Comment by bernerbrau** [comment]
And actually, this means the same thing applies for any action, really. You don't "roll will to defend" but you "feel the intrusion on your mind and try to resist". First rule of writing: "Show, don't tell." Fate says, "figure out what you want to do, then use the rules to do it." Same difference.

**Comment by kabobbob** [comment]
How was that?

**Comment by bernerbrau** [comment]
:+1:

**Comment by bernerbrau** [comment]
OK so, I had just come back from a run and my head wasn't totally there. Story wise, no problem. Mechanics wise that's definitely a create advantage because you're trying to change something about the scene (aspect "blinding fog"). GM decides if this is opposed or unopposed, and sets the difficulty if unopposed.

**Comment by bernerbrau** [comment]
So to be clear "Success with style" is +3 or more _over the opposition_, which is either passive (GM declares a difficulty) or active (opposing character rolls opposition). To my knowledge that hasn't been declared yet, but I may have missed something. 

Also, rather than a boost, success with style on a Create Advantage gives 2 free invokes on the created aspect. Failure creates the aspect but turns it against us. (So watch where you're pointing that spell!!)

https://fate-srd.com/fate-core/four-outcomes#succeed-with-style

**Comment by kabobbob** [comment]
Nope, you're right. I thought about it as I was editing it. I jumped the gun. (maybe he'll let it slide).
